### DISCUSSION TITLE
Upload historic data to pvoutput

### DISCUSSION THREAD

**Discussion by svendt** [SBFspot/SBFspot#666] [General]
Hello,

I installed SBFspot (Linux) and all seems to work. I have two SMA inverters configured (via bluetooth) and executed the following command to upload the historic 60-days data in the inverter to pvoutput (I'm a donator):

/usr/local/bin/sbfspot.3/SBFspot -v -ad62 -am12 -ae12 -sp0 -nocsv -finq

I noticed that only one of the two SMA inverters is uploading its historical data, is that expected and how can I force the other inverter to upload its data as well (or do I have to be more patiently and wait a couple of hours more)?

thanks!

**Comment by sbf-** [comment]
There are some things to check in the [wiki](https://github.com/SBFspot/SBFspot/wiki/No-uploads-to-PVOutput.org)

**Comment by svendt** [comment]
Thanks for the pointer. Seems that the time of inverter 2 was not good.
Had to change it with Sunny Explorer and all seems to be working now!
